charts/vaultwarden/Chart.yaml

```
@@ -13,5 +13,5 @@ maintainers:
   - name: guerzon
     email: guerzon@proton.me
     url: https://github.com/guerzon
-version: 0.22.1
+version: 0.23.0
```

```suggestion
version: 0.22.2
```

If this is a backward-compatible bug fix, perhaps the [semantic version](https://semver.org/) should be `0.22.2` instead of incrementing the minor version?